File: week15/week15-2.py
a=['HELLO','HOLA','HALLO','BONJOUR','CIAO','ZDRAVSTVUJTE']
b=['ENGLISH','SPANISH','GERMAN','FRENCH','ITALIAN','RUSSIAN']
c=1
while(now:=input())and now!='#':
	print(f'Case {c}: ',end='')
	if now not in a:print("UNKNOWN")
	else:print(b[a.index(now)])
	c+=1

File: week15/week15-2.cpp
#include <iostream>
#include <string>
using namespace std;
int main(){
	string hello;
	int t=1;
	while(cin>>hello) {
		if(hello=="#")break;	
		cout<<"Case "<<t<<": ";
		if(hello=="HELLO")cout<<"ENGLISH\n";
		else if(hello=="HOLA")cout<<"SPANISH\n";
		else if(hello=="HALLO")cout<<"GERMAN\n";
		else if(hello=="BONJOUR")cout<<"FRENCH\n";
		else if(hello=="CIAO")cout<<"ITALIAN\n";
		else if(hello=="ZDRAVSTVUJTE")cout<<"RUSSIAN\n";
		else cout<<"UNKNOWN\n";
		t++;
	}
}